optimade/server/mappers/entries.py

```diff
@@ -31,10 +31,10 @@ class BaseResourceMapper:
     def all_aliases(cls) -> Tuple[Tuple[str, str]]:
         return (
             tuple(
-                (CONFIG.provider["prefix"] + field, field)
+                (f"_{CONFIG.provider.prefix}_{field}", field)
                 for field in CONFIG.provider_fields.get(cls.ENDPOINT, [])
             )
-            + CONFIG.aliases.get(cls.ENDPOINT, ())
+            + tuple(CONFIG.aliases.get(cls.ENDPOINT, {}).items())
```

Comment:
Right! I was going to write a question here about why `CONFIG.aliases` was change to `Dict[str, dict]`, instead of being `Dict[str, tuple]`, but with the config file being a JSON and the new config setup, this makes sense why a `tuple` there would be difficult.

Reply:
Since `all_aliases` returned a tuple,  I was trying to not make too many breaking changes that would require more updates.